frontend/src/modules/batchactions/actions.js

```diff
@@ -2,6 +2,10 @@
 
 import api from 'core/api';
 
+import { actions as entitiesActions } from 'core/entities';
+import { actions as resourceActions } from 'core/resource';
+import { actions as statsActions } from 'core/stats';
```

Comment:
I don't very much like this way of importing things, I prefer the `import * as entities from 'core/entities';` form. I think it makes things clearer in the code when you read `entities.actions`. What do you think?

Reply:
Yeah I think it's better to enforce that we don't use module names as variable names in our code, to avoid confusion about what is what. Even though sometimes it makes naming a bit complex, I believe it is beneficial in the long term. It is not too big of a deal though.